### PR TITLE
ODF template update

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**

### CODE REVIEW

File: config-templates/redhat/odf-local/changelog.json [hunk 1]
```diff
@@ -3,6 +3,15 @@
         "4.12": {
             "status": "supported",
             "latest-revision": {
+                "revision": "2",
+                "release-date": "04 May 2023",
+                "cves": "CVE-2023-0361, CVE-2023-24536, CVE-2023-24537, CVE-2023-24538",
+                "golang": "1.19.8",
+                "ubi": "8.7-1107",
+                "new-features": [],
+                "fixes": ["LSO channel changed"]
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: If applicable, can you please add more detail about `LSO channel change`?
Like what it was earlier and whats difference now in new revision?

File: config-templates/redhat/odf-remote/changelog.json [hunk 2]
```diff
@@ -3,6 +3,15 @@
         "4.12": {
             "status": "supported",
             "latest-revision": {
+                "revision": "2",
+                "release-date": "04 May 2023",
+                "cves": "CVE-2023-0361, CVE-2023-24536, CVE-2023-24537, CVE-2023-24538",
+                "golang": "1.19.8",
+                "ubi": "8.7-1107",
+                "new-features": [],
+                "fixes": ["LSO channel changed"]
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json]: same here

**Review by mssachan — CHANGES_REQUESTED** [review]
I think you missed to update status of odf-remote 4.8 to `unsupported`

**Review by akgunjal — APPROVED** [review]
lgtm

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**